src/renderer/themes/lens-light.json

```diff
@@ -12,7 +12,7 @@
     "textColorPrimary": "#555555",
     "textColorSecondary": "#51575d",
     "textColorTertiary": "#555555",
-    "textColorAccent": "#333333",
+    "textColorAccent": "#222222",
```

Comment:
Why change?

Reply:
To match the delta on dark mode. Namely, `<Badge>`'s that are clickable now have their text change colour on hover. 

Without this change, on light mode, you can barely see the difference.

Reply:
What kind of side-effects this causes (if any)?

Reply:
The text on material UI tooltips on light-mode will be darker and so will the text for `FormSwitcher`. But both of those backgrounds are quite light. So I don't think it will be noticeable.